iceoryx_hoofs/platform/cmake/IceoryxPackageHelper.cmake

```diff
@@ -21,6 +21,14 @@
 # this also creates the config files
```

Comment:
I know that it is required to move this to the platform package but I think it is the wrong place. Maybe we need something like `iceoryx_core` which contains stuff like this but it should not be in platform. That package could also contain stuff like the building blocks for the logger or the error handler which are also not part of the platform itself.

Can you please create an issue for this?

Reply:
I created the issue: https://github.com/eclipse-iceoryx/iceoryx/issues/1633

I would agree to a `iceoryx_core` package when only cmake helper scripts and maybe C++/C code with absolutely no dependencies are stored. Here I would restrict it to not even use STL or libc.
I think this would be an interesting topic for the next coffee session